musicbrainz-dockerfile/DBDefs.pm

```diff
@@ -52,16 +52,15 @@ MusicBrainz::Server::DatabaseConnectionFactory->register_databases(
     # How to connect when we need read-write access to the database
     READWRITE => {
         database    => "musicbrainz_db",
-        schema          => "musicbrainz",
+        schema      => "musicbrainz",
         username    => "$ENV{DB_ENV_POSTGRES_USER}",
-        password        => "$ENV{DB_ENV_POSTGRES_PASSWORD}",
-        host            => "$ENV{DB_PORT_5432_TCP_ADDR}",
-        port            => "$ENV{DB_PORT_5432_TCP_PORT}",
+        password    => "$ENV{DB_ENV_POSTGRES_PASSWORD}",
+        host        => "db",
```

Comment:
I see thanks for the info, is there a doc you can point me to with more details on this?